zipline/finance/performance/tracker.py

```diff
@@ -189,7 +199,9 @@ def get_portfolio(self, performance_needs_update):
         if performance_needs_update:
             self.update_performance()
             self.account_needs_update = True
-        return self.cumulative_performance.as_portfolio()
+        portfolio = self.cumulative_performance.as_portfolio()
+        portfolio.current_date = self._current_session
```

Comment:
Alternatively, we might want to pass a callback to the portfolio to get the current datetime.  We do this in a bunch of other places for objects that are tied to the context of the simulation.